staging/src/kubevirt.io/client-go/log/log.go

```diff
@@ -227,13 +283,31 @@ func (l FilteredLogger) Object(obj LoggableObject) *FilteredLogger {
 	return &l
 }
 
+func (l FilteredLogger) ObjectRef(obj *v1.ObjectReference) *FilteredLogger {
```

Comment:
I think ObjectRef got removed at some point and is not used, can that be?

Reply:
Yes, it was removed in 53a3fced325c6448aae451191ba843d384ff3b52
I've removed it now as well.